tests/field_defaults/tests.py

```diff
@@ -48,7 +48,12 @@ def test_field_database_defaults_returning(self):
 
         self.assertIsInstance(a.id, int)
         self.assertEqual(a.headline, "Default headline")
-        self.assertLess((a.pub_date - now).seconds, 5)
+
+        # Work around a flakiness in the Oracle setup
+        if connection.vendor == "oracle" and now > a.pub_date:
+            self.assertLess((now - a.pub_date).seconds, 5)
```

Comment:
Perhaps a better fix would be to just skip this test on Oracle.

Reply:
IMO using `self.assertGreaterEqual()` is enough.

Reply:
I'm not sure how that helps. The problem I had here was that sometimes `now` was more than `pub_date` when it was expected to be less. I think it has something to do with clocks not being perfectly synchronised between python and the database.

Reply:
That's possible :thinking:. I encountered the same thing in MySQL several times. Let's leave only `IsInstance` assertion.